File: api-service/src/main/java/org/api/data/UserAppkey.java
package org.api.data;


import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode
public class UserAppkey {

    private String appkey;

    private String userid;
    private int usertype;

    @Override
    public String toString() {
        return "UserAppkey{" +
                "appkey='" + appkey + '\'' +
                ", userid='" + userid + '\'' +
                ", usertype=" + usertype +
                '}';
    }
}

File: api-service/src/main/kotlin/org/api/appkey/UserAppkey.kt
package org.api.appkey


class UserAppkey {
    var appkey: String? = null
    var userid: String? = null
    var usertype = 0

    constructor() {

    }

    constructor(appkey: String?, userid: String?, usertype: Int) {
        this.appkey = appkey
        this.userid = userid
        this.usertype = usertype
    }

}